app/src/main/res/values/strings.xml

```diff
@@ -1,91 +1,210 @@
+  <string name="admin_pin_activity_admin_pin_input_hint_text">New 5-Digit PIN</string>
+  <string name="admin_pin_activity_admin_pin_input_confirm_pin_text">Confirm 5-Digit PIN</string>
+  <string name="admin_pin_activity_admin_pin_length_error_text">Your PIN should be 5 digits long.</string>
+  <string name="admin_pin_activity_wrong_confirm_admin_pin_error_text">Please make sure that both PINs match.</string>
+  <string name="admin_pin_activity_label_text">Authorise to add profiles</string>
+  <!-- AdministratorControlsFragment -->
+  <string name="administrator_controls_fragment_test_activity_label">Administrator Controls Fragment Test Activity</string>
+  <string name="administrator_controls_general_view_administrator_controls_general_label_text">General</string>
+  <string name="administrator_controls_general_view_administrator_edit_account">Edit account</string>
+  <string name="administrator_controls_profile_view_administrator_controls_profile_management_label_text">Profile Management</string>
+  <string name="administrator_controls_profile_view_administrator_edit_profiles">Edit profiles</string>
+  <string name="administrator_controls_download_permissions_view_label_text">Download Permissions</string>
+  <string name="administrator_controls_download_permissions_view_administrator_controls_update_on_wifi_title">Download and update only on Wi-fi</string>
+  <string name="administrator_controls_download_permissions_view_administrator_controls_update_on_wifi_message">Topics will be downloaded and updated only on Wi-fi. Any downloads or updates of cellular data will be queued.</string>
+  <string name="administrator_controls_download_permissions_view_administrator_controls_auto_update_title">Automatically update topics</string>
+  <string name="administrator_controls_download_permissions_view_administrator_controls_auto_update_message">Downloaded topics that have new content available will automatically update.</string>
+  <string name="administrator_controls_app_information_view_administrator_controls_app_information_label_text">App Information</string>
+  <string name="administrator_controls_app_information_view_app_version">App Version</string>
+  <string name="administrator_controls_account_actions_view_administrator_controls_account_actions_label_text">Account Actions</string>
+  <string name="administrator_controls_account_actions_view_log_out_text">Log Out</string>
+  <string name="administrator_controls_view_log_out_dialog_fragment_cancel_button">Cancel</string>
+  <string name="administrator_controls_view_log_out_dialog_fragment_okay_button">Ok</string>
+  <string name="administrator_controls_view_log_out_dialog_fragment_message">Are you sure you want to log out of your profile?</string>
+  <!-- AppVersionFragment -->
+  <string name="administrator_controls_app_version_activity_app_version_name">App Version %s</string>
```

Comment:
For this & the block above: are these strings only being used by fragments? We could instead name them based on the fragment like we do for activities. WDYT?

Reply:
> For this & the block above: are these strings only being used by fragments? We could instead name them based on the fragment like we do for activities. WDYT?

@BenHenning Can you please evaluate this. 
Are you talking about the heading `<!-- AppVersionFragment -->` to `<!-- AppVersionActivity -->`

Reply:
Done.